lrpar/src/lib/parser.rs

```diff
@@ -233,8 +233,8 @@ where
     /// `end_laidx` *must* be set to `laidx + 1` in order that the parser doesn't skip the real
     /// lexeme at position `laidx`.
     ///
-    /// Return `true` if the parse reached an accept state (i.e. all the input was consumed,
-    /// possibly after making repairs) or `false` (i.e. some of the input was not consumed, even
+    /// Return `Some(v)` if the parse reached an accept state (i.e. all the input was consumed,
```

Comment:
Do we need to mention what `v` is?

Reply:
Good question. I don't really know *what* to call the variable, as it depends on `ActionT`. So I ended up giving it a semi-meaningless name... I'm open to suggestions!

Reply:
Ah, I see. In that case maybe `value` is a bit more meaningful without being too meaningful?